project/src/test/kotlin/org/ionproject/core/calendar/CalendarControllerTest.kt

```diff
@@ -16,84 +16,22 @@ import org.junit.jupiter.api.Assertions
- ...
-CREATED:20200101T163530Z
-DUE:20200705T235900Z
-CATEGORIES;LANGUAGE=pt-PT:LaboratÃ³rio
-CATEGORIES;LANGUAGE=en-US:Laboratory
-CATEGORIES;LANGUAGE=en-GB:Laboratory
-END:VTODO
-BEGIN:VEVENT
-UID:2c
-DTSTAMP:20200101T163530Z
-SUMMARY;LANGUAGE=en-US:1st Exam WAD
-DESCRIPTION;LANGUAGE=en-US:Normal season exam for WAD-1920v
-CREATED:20200101T163530Z
-CATEGORIES;LANGUAGE=pt-PT:Exame
-CATEGORIES;LANGUAGE=en-US:Exam
-CATEGORIES;LANGUAGE=en-GB:Exam
-DTSTART:20200619T180000Z
-DTEND:20200619T193000Z
-LOCATION:Room A.2.10
-END:VEVENT
-BEGIN:VEVENT
-UID:2d
-DTSTAMP:20200101T163530Z
-SUMMARY;LANGUAGE=en-US:2nd Exam WAD
-DESCRIPTION;LANGUAGE=en-US:Second season exam for WAD-1920v
-CREATED:20200101T163530Z
-CATEGORIES;LANGUAGE=pt-PT:Exame
-CATEGORIES;LANGUAGE=en-US:Exam
-CATEGORIES;LANGUAGE=en-GB:Exam
-DTSTART:20200701T100000Z
-DTEND:20200701T123000Z
-LOCATION:Room A.2.10
-END:VEVENT
-END:VCALENDAR"""
-
-    val calendarByClassSection = """BEGIN:VCALENDAR
+    companion object {
+        val courseID = 2
+        val calTerm = "1920v"
+        val classSection = "LI61D"
+        val componentIDClassSection = "15"
+        val componentIdClass = "1f"
+
+        val calendarByClass =
+            """
+                BEGIN:VCALENDARPRODID:/v0/courses/2/classes/1920vVERSION:2.0BEGIN:VTODOUID:1fDTSTAMP:20200101T163530ZSUMMARY;LANGUAGE=en-US:[WAD]:Assignment#1DESCRIPTION;LANGUAGE=en-US:Thefirstassignment.ThegoalistoimplementanHTTPAPI...ATTACH:https://tools.ietf.org/html/rfc7231CREATED:20200101T163530ZDUE:20200419T235900ZCATEGORIES;LANGUAGE=pt-PT:LaboratórioCATEGORIES;LANGUAGE=en-US:LaboratoryCATEGORIES;LANGUAGE=en-GB:LaboratoryEND:VTODOBEGIN:VTODOUID:20DTSTAMP:20200101T163530ZSUMMARY;LANGUAGE=en-US:[WAD]:Assignment#2DESCRIPTION;LANGUAGE=en-US:Thesecondassignment.ThegoalistoimplementaWebClientfortheAPI...CREATED:20200101T163530ZDUE:20200525T235900ZCATEGORIES;LANGUAGE=pt-PT:LaboratórioCATEGORIES;LANGUAGE=en-US:LaboratoryCATEGORIES;LANGUAGE=en-GB:LaboratoryEND:VTODOBEGIN:VTODOUID:21DTSTAMP:20200101T163530ZSUMMARY;LANGUAGE=en-US:[WAD]:Assignment#3DESCRIPTION;LANGUAGE=en-US:Thethirdandfinalassignment.Wrappingitup...CREATED:20200101T163530ZDUE:20200705T235900ZCATEGORIES;LANGUAGE=pt-PT:LaboratórioCATEGORIES;LANGUAGE=en-US:LaboratoryCATEGORIES;LANGUAGE=en-GB:LaboratoryEND:VTODOBEGIN:VEVENTUID:2cDTSTAMP:20200101T163530ZSUMMARY;LANGUAGE=en-US:1stExamWADDESCRIPTION;LANGUAGE=en-US:NormalseasonexamforWAD-1920vCREATED:20200101T163530ZCATEGORIES;LANGUAGE=pt-PT:ExameCATEGORIES;LANGUAGE=en-US:ExamCATEGORIES;LANGUAGE=en-GB:ExamDTSTART:20200619T180000ZDTEND:20200619T193000ZLOCATION:RoomA.2.10END:VEVENTBEGIN:VEVENTUID:2dDTSTAMP:20200101T163530ZSUMMARY;LANGUAGE=en-US:2ndExamWADDESCRIPTION;LANGUAGE=en-US:SecondseasonexamforWAD-1920vCREATED:20200101T163530ZCATEGORIES;LANGUAGE=pt-PT:ExameCATEGORIES;LANGUAGE=en-US:ExamCATEGORIES;LANGUAGE=en-GB:ExamDTSTART:20200701T100000ZDTEND:20200701T123000ZLOCATION:RoomA.2.10END:VEVENTEND:VCALENDAR
```

Comment:
Can we avoid having this long line and keep the expected value in multiple lines?